app/renderer/js/utils/system-util.ts

```diff
@@ -1,12 +1,17 @@
 'use strict';
 
-const {app} = require('electron').remote;
+import { remote } from 'electron';
+const { app } = remote;
+import * as os from 'os';
 
-const os = require('os');
+let instance: any = null;
 
-let instance = null;
+export interface systemUtil {
+	connectivityERR: string[],
+	userAgent: any
+}
 
-class SystemUtil {
+export class systemUtil {
```

Comment:
@priyank-p @vsvipul I don't think export is needed here, can be removed.

Reply:
Fixed this.